bool autofocus
uint8 focusValue
---
uint8 result
